web-app/helpers/certificate-designs.ts

```diff
@@ -53,24 +55,33 @@ function wrapText(canvas: Canvas, text: string, x: number, y: number, maxWidth:
       line = testLine.trim();
     }
   }
+  context.fillStyle = fillStyle;
+  context.font = font;
   context.fillText(line, x, y2);
 }
 
 // eslint-disable-next-line max-lines-per-function
-export async function populateDeveloperCert(canvas: Canvas, details: any) {
-  console.log('populateDeveloperCert', { details });
+export async function populateCerts(canvas: Canvas, details: any) {
+  console.log('populateCerts', { details });
```

Comment:
In this function name, let's use `populateCert` singular rather than plural `Certs` since it's only working on one cert at a time.

Reply:
but you didn't update the console log ;-)

Reply:
whoops 😅
done